Tensile/TensileCreateLibrary.py

```diff
@@ -1015,6 +1015,7 @@ def TensileCreateLibrary():
   argParser.add_argument("--embed-library-key",      dest="EmbedLibraryKey", default=None,
                          help="Access key for embedding library files.")
   argParser.add_argument("--version", help="Version string to embed into library file.")
+  argParser.add_argument("--yaml", action="store_true", help="Use YAML format for writing")
```

Comment:
The format should also be able to come from `globalParameters` – this will enable using either format during tuning and allow testing both formats in CI.